src/clj/puppetlabs/services/jruby/jruby_core.clj

```diff
@@ -142,43 +142,43 @@
 ;;; Public
 
 (schema/defn ^:always-validate
-  initialize-config :- jruby-schemas/JRubyPuppetConfig
+  initialize-config :- jruby-schemas/JRubyConfig
   [config :- {schema/Keyword schema/Any}]
-  (-> (get-in config [:jruby-puppet])
+  (-> (get-in config [:jruby])
```

Comment:
Yeah, I'm good with leaving it how @rlinehan has it here.  I'm also assuming the API for this library, assuming we get rid of any TK tie-ins, would have "puppet" references removed where appropriate.